src/main/java/com/rcv/ResultsWriter.java

```diff
@@ -445,13 +452,13 @@ private void generateSummaryJson(Map<Integer, Map<String, BigDecimal>> roundTall
     // config will contain contest configuration info
     HashMap<String, Object> configData = new HashMap<>();
     // add config header info
-    configData.put("Contest", config.getContestName());
-    configData.put("Jurisdiction", config.getContestJurisdiction());
-    configData.put("Office", config.getContestOffice());
-    configData.put("Date", config.getContestDate());
-    configData.put("Threshold", winningThreshold.toString());
+    configData.put("contest", config.getContestName());
```

Comment:
At one point I tried mapping the summary json into a java object using Jackson for comparison as a java object.  I switched to lower-case names here to match our java variable naming conventions.  In the end I decided that the line-by-line file compare method was better suited for our purpose because it was less code and wouldn't need to be updated whenever the json format changes.